src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -82,10 +87,27 @@ export default class List extends SfdxCommand {
     }
 
     private showDeviceList(list: any[]) {
+        let duration: number = 0;
+
+        const obs = new PerformanceObserver((items) => {
```

Comment:
Minor: we should add `observer` to this signature, and do an `observer.disconnect()` at the end, so it doesn't hang around longer than it needs to.